updater/lib/dependabot/updater/group_update_refreshing.rb

```diff
@@ -41,6 +41,9 @@ def upsert_pull_request_with_error_handling(dependency_change, group)
         end
       rescue StandardError => e
         error_handler.handle_job_error(error: e, dependency_group: dependency_snapshot.job_group)
+      ensure
+        # record metrics for the ecosystem
+        service.record_ecosystem_meta(dependency_snapshot.ecosystem)
       end
```

Comment:
**Review Tip:** Send metrics for group refresh operation after update process (success, or fail)